rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -999,6 +989,25 @@ def _cleanup_cluster(rosa_cmnd, cluster_name, mgmt_cluster_name, my_path, my_uui
         es_ignored_metadata = ""
         common._index_result(es, index, metadata, es_ignored_metadata, index_retry)
 
+# This func() will label required nodes, requires top-level directory to kubeconfig,
+# cluster_name  & worker_name, worker_label defaults to 
+def _label_nodes(my_path, cluster_name, worker_name, worker_label):
```

Comment:
This will not reliably work in ROSA. The labels on nodes are generally reconciled with what is in OCM. Labeling of nodes needs to happen via the rosa cli. IMHO we should assume the labels are done correctly at this point and just target those labels if the option is passed.

Reply:
Russ as discussed - will tackle in next PR